libs/barista-components/card/src/_card-theme.scss

```diff
@@ -1,26 +1,15 @@
-@import '../../core/src/theming/theming';
+* > .dt-theme-dark > * {
```

Comment:
I believe this selector could look a little different. This one seems odd. Can you explain your reasoning behind this?

In the button, this is also sufficient.
https://github.com/dynatrace-oss/barista/blob/4f3f47bc957d328611eb0ca1da95fc38cf526b11/libs/barista-components/button/src/_button-theme.scss#L54-L70